src/main/java/net/fabricmc/example/ExampleModClient.java

```diff
@@ -1,24 +1,22 @@
-package me.luligabi.tutorial;
+package net.fabricmc.example;
 
 import net.fabricmc.api.ClientModInitializer;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.fabricmc.fabric.api.client.particle.v1.ParticleFactoryRegistry;
-import net.fabricmc.fabric.api.event.client.ClientSpriteRegistryCallback;
 import net.minecraft.client.particle.FlameParticle;
-import net.minecraft.screen.PlayerScreenHandler;
-import net.minecraft.util.Identifier;
 
 @Environment(EnvType.CLIENT)
 public class ExampleModClient implements ClientModInitializer {
 
     @Override
     public void onInitializeClient() {
-        ClientSpriteRegistryCallback.event(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE).register(((atlasTexture, registry) -> {
-            registry.register(new Identifier("modid", "particle/green_flame"));
-        }));
-
+        /* Registers our particle client-side.
+         * First argument is our particle's instance, created previously on ExampleMod.
+         * Second argument is the particle's factory. The factory controls how the particle behaves.
+         * In this example, we'll use FlameParticle's Factory. */
         ParticleFactoryRegistry.getInstance().register(ExampleMod.GREEN_FLAME, FlameParticle.Factory::new);
```

Comment:
Does only this work? I recall there being some performance-related change to how textures are loaded on 1.19.3 (hence the removal of `ClientSpriteRegistryCallback`) but there's no replacement for it whatsover? I'd like to test this but I won't be able to until friday.

Reply:
I'm not sure if there is a replacement for `ClientSpriteRegistryCallback`.
The way I have the whole mod currently set up is the only way I could get particles working. (I'm really hoping I understand what you're saying.)

Reply:
> I'm not sure if there is a replacement for `ClientSpriteRegistryCallback`. The way I have the whole mod currently set up is the only way I could get particles working. (I'm really hoping I understand what you're saying.)

I meant that previously we needed the callback since the game wouldn't load particles outside of their own namespace. This has changed with 1.19.3, meaning there's no need for what the callback did back then.